packages/web-app-files/src/components/Shares/SharedWithMeSection.vue

```diff
@@ -284,15 +284,9 @@ export default defineComponent({
         onExit: debounced.cancel
       })
     },
-    getShowSynchedIcon(resource: Resource) {
+    getShowSynchedIcon(resource: ShareResource) {
       return resource.status === ShareStatus.accepted
```

Comment:
Maybe now is a good time to rename ` .status` to something semantically more meaningful?

Reply:
> Maybe now is a good time to rename  .status to something semantically more meaningful?

Yes, actually we should get rid of the `status` completely in favour of the new `syncEnabled` since we won't have statuses anymore 😄